phpunit/class-block-fixture-test.php

```diff
@@ -7,6 +7,12 @@
 
 class Block_Fixture_Test extends WP_UnitTestCase {
 
+	public function filter_allowed_html( $tags ) {
+		$tags['form']['class']   = true;
+		$tags['form']['enctype'] = true;
+		return $tags;
+	}
```

Comment:
The `wp_kses_post()` function does not allow `form` tag [by default](https://github.com/WordPress/wordpress-develop/blob/0ce733c78f86c4ea06dd0dde329fcca571179f50/src/wp-includes/kses.php#L153-L167). So we need to allow attributes to be included in the serialized form block.